superset/viz.py

```diff
@@ -681,9 +687,13 @@ def get_data(self, df):
         if aggfunc == 'sum':
             aggfunc = lambda x: x.sum(min_count=1)  # noqa: E731
 
+        groupby = self.form_data.get('groupby')
+        columns = self.form_data.get('columns')
+        if self.form_data.get('transpose_pivot'):
+            groupby, columns = columns, groupby
         df = df.pivot_table(
-            index=self.form_data.get('groupby'),
-            columns=self.form_data.get('columns'),
+            index=groupby,
```

Comment:
Would using `df = df.T` instead work?

Reply:
https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.DataFrame.T.html

Reply:
Hi, Max, if we use df = df.T transpose pivot table, the metric columns will also rotate, 
this can cause problems for rendering dataframe html, such as columns out of cellindex.(Because there will be spaces cell on the column)

### original table
![image](https://user-images.githubusercontent.com/2016594/56542802-24526e00-65a2-11e9-923d-c42e6a8b995b.png)

### when df.T
![image](https://user-images.githubusercontent.com/2016594/56542819-2caaa900-65a2-11e9-9902-5881c1766455.png)

Reply:
If the metric columns must be rotated, may need to modify some superset-ui-chart

Reply:
@mistercrunch Do you have any suggestions?